MRI2CBCT_CLI/MRI2CBCT_CLI_utils/resample_create_csv.py

```diff
@@ -1,6 +1,12 @@
 import SimpleITK as sitk
 import os
-import pandas as pd
+
+try : 
+    import pandas as pd
+except ImportError:
+    from slicer.util import pip_install
+    pip_install("pandas")
```

Comment:
Why not add this to `install_function()`?

Reply:
I don't know, I just forgot. I'm gonna do it thanks !

Reply:
I've made the change !